tutorials/Numerics/TimeStepping/tutorial_risingbubble_config.jl

```diff
@@ -0,0 +1,316 @@
+    δ_χ = SVector{ntracers, FT}(1, 2, 3, 4)
+    ## To assemble `AtmosModel` with no tracers, set `tracers = NoTracers()`.
+
+    ## The model coefficient for the turbulence closure is defined via the
+    ## [CLIMAParameters
+    ## package](https://CliMA.github.io/CLIMAParameters.jl/latest/) A reference
+    ## state for the linearisation step is also defined.
+    T_surface = FT(300)
+    T_min_ref = FT(0)
+    T_profile = DryAdiabaticProfile{FT}(param_set, T_surface, T_min_ref)
+    ref_state = HydrostaticState(T_profile)
+
+    ## Here we assemble the `AtmosModel`.
+    _C_smag = FT(C_smag(param_set))
+    model = AtmosModel{FT}(
+        AtmosLESConfigType,                            ## Flow in a box, requires the AtmosLESConfigType
+        param_set;                                     ## Parameter set corresponding to earth parameters
+        init_state_prognostic = init_risingbubble!,    ## Apply the initial condition
+        ref_state = ref_state,                         ## Reference state
+        turbulence = SmagorinskyLilly(_C_smag),        ## Turbulence closure model
+        moisture = DryModel(),                         ## Exclude moisture variables
+        source = (Gravity(),),                         ## Gravity is the only source term here
+        tracers = NTracers{ntracers, FT}(δ_χ),         ## Tracer model with diffusivity coefficients
+    )
+
+    ## Finally, we pass a `Problem Name` string, the mesh information, and the
+    ## model type to  the [`AtmosLESConfiguration`] object.
+    config = ClimateMachine.AtmosLESConfiguration(
+        "DryRisingBubble",       ## Problem title [String]
+        N,                       ## Polynomial order [Int]
+        resolution,              ## (Δx, Δy, Δz) effective resolution [m]
+        xmax,                    ## Domain maximum size [m]
+        ymax,                    ## Domain maximum size [m]
+        zmax,                    ## Domain maximum size [m]
+        param_set,               ## Parameter set.
+        init_risingbubble!,      ## Function specifying initial condition
+        ## solver_type = ode_solver,## Time-integrator type
+        model = model,           ## Model type
+    )
+    return config
+end
+
+#md # !!! note
+#md #     `Keywords` are used to specify some arguments (see appropriate source
+#md #     files).
+
+# ## Diagnostics
+# Here we define the diagnostic configuration specific to this problem.
+function config_diagnostics(driver_config)
+    interval = "10000steps"
+    dgngrp = setup_atmos_default_diagnostics(
+        AtmosLESConfigType(),
+        interval,
+        driver_config.name,
+    )
+    return ClimateMachine.DiagnosticsConfiguration([dgngrp])
+end
+
+function run_simulation(ode_solver, CFL, timeend)
+    ## These are essentially arguments passed to the
+    ## [`config_risingbubble`](@ref config-helper) function.  For type
+    ## consistency we explicitly define the problem floating-precision.
+    FT = Float64
```

Comment:
```suggestion
function run_simulation(ode_solver, CFL::FT, timeend::FT) where {FT}
```
We could alternatively grab it from `CFL` and `timeend`